requirement/dev.txt

```diff
@@ -1,4 +1,5 @@
 -r include/build.txt
 bumpversion==0.5.3
+git+https://github.com/sprin/markdown-inline-graphviz.git@master # pypi version doesn't work on python3 yet
```

Comment:
So, yeah, they haven't tagged this fix, but it would probably be a good idea to pin to the commit, rather than to whatever they push to `master`, _e.g._:

```suggestion
git+https://github.com/sprin/markdown-inline-graphviz.git@9664863b3002d88243c9ee5e14c195e037e54618  # pypi version doesn't work on python3 yet
```